fractions/microprofile/microprofile/README.adoc

```diff
@@ -1,9 +1,9 @@
 [id='microprofile']
-= MicroProfile
+= Eclipse MicroProfile
```

Comment:
Same here. We can add 1 sentence below, something like `This fraction provides support for the entire Eclipse MicroProfile platform.`